pipeline/outputs/stackdriver_special_fields.md

```diff
@@ -1,19 +1,29 @@
 # Stackdriver Special fields
```

Comment:
Why do we need to split this out into a separate doc @braydonk ?

Reply:
I don't know, this was done well before my time by another Google employee who doesn't work in this area anymore. I'll check with my team and see if anyone would mind merging it into the main doc

Reply:
Appreciate it mate, I was just thinking it might be easier to merge into one but if not I can merge